api/main.py

```diff
@@ -1,20 +1,57 @@
 from fastapi import FastAPI, Request, status
 from fastapi.encoders import jsonable_encoder
-from fastapi.exceptions import RequestValidationError
+from fastapi.exceptions import RequestValidationError, HTTPException
 from fastapi.responses import JSONResponse
 # from controller import Controller
+from starlette.responses import PlainTextResponse
+
+from object_store import ObjectStore
+import boto3
 
 import pickle
-import json
+import json, os
+import logging
 
 import numpy
 
+COMPLETED_BUCKET = os.getenv("COMPLETED_BUCKET")
+PENDING_BUCKET = os.getenv("PENDING_BUCKET")
+
+# Initialize ObjectStore
+try:
+    ob = ObjectStore()
+except Exception as ex:
+    logging.error("Error is -", ex)
+
+
 app = FastAPI()
 
 @app.get("/")
 async def homepage(request: Request):
     return("Welcome to Async Job Framework")
 
+
+@app.get("/getResult/{job_id}")
```

Comment:
Use `post` instead of `get`

Reply:
With a json body like - 
{
 job_id: 1234
}
Something like this?